src/sdraw.c

```diff
@@ -1,7 +1,8 @@
 #include "main.h"
 
 const int SPEEDS[] =  {658, 478, 378, 298, 228, 178, 138, 108, 88};
-static SDL_Surface * screen = NULL;
+static SDL_Surface *screen = NULL;
```

Comment:
Why do you need this global variable if you call `SDL_GetWindowSurface` before each drawing call?

Reply:
Any reason we can't keep the global and only set it at init/resize time - and remove the local?